### ISSUE TITLE
clone site displays no media

### ISSUE THREAD

**Issue by farahalibrahim** [recloudstream/cloudstream#1361]
### Steps to reproduce

1. From Arabico repository [https://raw.githubusercontent.com/aymanbest/Arabico/main/repo.json](url) clone "3isk v2" using clone link [https://eshqo.com/](url)
2. 

### Expected behavior

The original domain is blocked in my region & need VPN to work. "Cl one Site" is supposed to offer another version of existing provider whose domain in blocked by a mirror link working & not blocked.

provided/Used mirror link works without VPN on browser, so cloned site must so too...

### Actual behavior

cloned website just like the original provider shows no content. Although website works in browser
![Screenshot_20241001_223327_CloudStream Beta](https://github.com/user-attachments/assets/cdeffe07-3a14-45a9-833b-81fceee6bf25)
![Screenshot_20241001_223504_CloudStream Beta](https://github.com/user-attachments/assets/763e2a58-0701-439f-bf52-f0532093b1f6)
![Screenshot_20241001_223522_Chrome](https://github.com/user-attachments/assets/72aa83db-b291-4c99-b5b2-5bc8cdad6df5)

### Cloudstream version and commit hash

4.4.0 f4fd303

### Android version

Android 14

### Acknowledgements

- [X] I am sure my issue is related to the app and **NOT some extension**.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to pre-release version **[Latest](https://github.com/recloudstream/cloudstream/releases)**.
- [X] I will fill out all of the requested information in this form.
